src/main/scala/iog/psg/cardano/CardanoApiCodec.scala

```diff
@@ -364,10 +364,26 @@ object CardanoApiCodec {
                   )
 
   @ConfiguredJsonCodec(decodeOnly = true)
-  case class TimedBlock(
-                         time: ZonedDateTime,
-                         block: Block
-                       )
+  final case class TimedBlock(
+                               time: ZonedDateTime,
+                               @deprecated("block parameter is removed in latest API")
```

Comment:
Whic version was it removed in? It won't always be the "latest"

Reply:
(I think it's tradition to use 'removed since' or something like that?)

Reply:
ah yes, its hard to say since even what day, looks like this change is only in latest tag.
maybe **since november 2020** ?

Reply:
will check on docker hub again, and back here soon

Reply:
Don't that have the tag in the code repo? https://github.com/input-output-hk/cardano-wallet/releases/tag/v2020-11-17
So use the tag you tested against.

Reply:
Ok Ive put **v2020-11-17** insdie this deprecation warning then